src/components/ChatBeginingText.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import Str from 'expensify-common/lib/str';
+import {withOnyx} from 'react-native-onyx';
+import _ from 'underscore';
+import styles from '../styles/styles';
+import Text from './Text';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import compose from '../libs/compose';
+import {getPersonalDetailsForLogins} from '../libs/OptionsListUtils';
+import ONYXKEYS from '../ONYXKEYS';
+
+const propTypes = {
+    /** Weather it is a default Chat Room */
+    isDefaultChatRoom: PropTypes.bool,
+
+    /** The report currently being looked at */
+    report: PropTypes.oneOfType([PropTypes.object]),
+
+    ...withLocalizePropTypes,
+
+};
+
+const defaultProps = {
+    report: {},
+    isDefaultChatRoom: false,
+};
+
+const ChatBeginingText = ({
+    isDefaultChatRoom, report, translate, personalDetails, toLocalPhone,
+}) => {
+    const participants = lodashGet(report, 'participants', []);
+    const isMultipleParticipant = participants.length > 1;
+    const displayNamesWithTooltips = _.map(
+        getPersonalDetailsForLogins(participants, personalDetails),
+        ({
+            displayName, firstName, login, pronouns,
+        }) => {
+            const displayNameTrimmed = Str.isSMSLogin(login) ? toLocalPhone(displayName) : displayName;
+
+            return {
+                displayName: (isMultipleParticipant ? firstName : displayNameTrimmed) || Str.removeSMSDomain(login),
+                tooltip: Str.removeSMSDomain(login),
+                pronouns,
+            };
+        },
+    );
+    const chatUsers = isDefaultChatRoom ? [{displayName: report.reportName}] : displayNamesWithTooltips;
+
+    return (
+        <Text style={[styles.mt3, styles.w70, styles.textAlignCenter]}>
+            <Text>
+                {isDefaultChatRoom ? `${translate('reportActionsView.beginingOfChatHistroyPrivate')} ` : `${translate('reportActionsView.beginingOfChatHistroy')} `}
```

Comment:
Please Wrap this in the next lines. https://github.com/Expensify/App/blob/main/STYLE.md#inline-ternarys